src/ScreenEdit.cpp

```diff
@@ -1526,6 +1528,21 @@ void ScreenEdit::Init()
 		SetDirty(true);
 	}
 
+
+	// There are two conditions for centering the notefield.
+	// 1. We're in EditMode
+	// 2. We're in PracticeMode, the preference is enabled,
+	//      multiplayer is not enabled, and the theme allows it.
+	ThemeMetric<bool> allow_center;
+	allow_center.Load("ScreenGameplay", "AllowCenter1Player");
+	bool center_enabled = (kCenter1Player.GetPreferenceByName("Center1Player")->Get()
```

Comment:
Let's stick to the same conditions listed out in ScreenGameplay:

https://github.com/itgmania/itgmania/blob/5d4f9dcb07493ed4c51d6be23e9b41978162305a/src/ScreenGameplay.cpp#L880

Reply:
Sure, I've changed the style check.

I've tentatively added the `m_PlayMode` checks as well, though do we really think they'll ever be anything but `PLAY_MODE_REGULAR`? I guess with some theme hackery, PracticeMode could be accessed in any PlayMode. Thinking about it more, it might even be nice to add `PLAY_MODE_PRACTICE` to distinguish the PracticeMode more concretely, but again that's outside scope for now.